_api-reference/index-apis/recover.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: default
+title: Recover index
+parent: Index APIs
+nav_order: 62
+---
+
+# Recover an index 
+Introduced 1.0
+{: .label .label-purple }
+
+The Recover index API gives information about any completed or ongoing shard recoveries for one or more indexes. If a data stream is listed, the API returns information about that data stream's backing index. 
+
+Shard recovery is the process of starting a shard copy for restoring a primary shard from a snapshot or syncing a replica shard. Once a shard recovery is complete, the recovered shard becomes available for searching and indexing operations.
+
+Shard recovery occurs automatically in the following scenarios:
+
+- Node startup, known as a local store recovery.
+- Replication of a primary shard.
+- Relocation of a shard to a different node within the same cluster.
+- Restoration of a snapshot.
+- Clone, shrink, or split operations.
+
+The Recover Index API provides information only about completed recoveries for shard copies that currently exist in the cluster. It reports only the most recent recovery for each shard copy and does not include historical information about previous recoveries or information about recoveries of shard copies that no longer exist. Consequently, if a shard copy completes a recovery and is subsequently relocated to a different node, the information about the original recovery will not be displayed in the Recover Index API.
+
+
+## HTTP and Path methods
+
+```
+GET /_recovery
+GET /<index-name>/recovery/
+```
+
+## Path parameters
+
+Parameter | Type | Description 
+:--- | :--- 
+`index-name` | String | The name(s) of the index(es) you want recovery information about.  
```

Comment:
The path parameter `index-name` could be index, data stream, or alias, and supports wildcards. We'd better have an uniform way to handle this kind of parameter, because I see the existing documentation of other APIs such as `search`, `count` and `refresh` also have this problem.